api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -443,4 +445,14 @@ public static class App {
   public static class E2ETestUserConfig {
     public List<String> testUserEmails;
   }
+
+  // Configs for AoU and Terra Terms of Service
+  public static class TermsOfServiceConfig {
+    // which All of Us Terms of Service is considered current
+    public int latestAouVersion;
```

Comment:
This is also new.  I figured I'd take this opportunity to configify here.  But LMK if I should remove it.